packages/babel-plugin-transform-block-scoping/src/tdz.js

```diff
@@ -4,9 +4,9 @@ function getTDZStatus(refPath, bindingPath) {
   const executionStatus = bindingPath._guessExecutionStatusRelativeTo(refPath);
 
   if (executionStatus === "before") {
-    return "inside";
-  } else if (executionStatus === "after") {
     return "outside";
+  } else if (executionStatus === "after") {
+    return "inside";
```

Comment:
I think that "inside" and "outside" were swapped? To me "inside" means "inside the temporal dead zone", so "the usage is after the lexical declaration".

Reply:
Yeah looking at the code it seems that "inside" was meant as "inside valid Zone". It is easier to understand for me now after swapping.